### ISSUE TITLE
parallel: cleanup

### ISSUE THREAD

**Issue by dtzWill** [NixOS/nixpkgs#46476]
## Issue description

Noticed build failure in derivation using parallel (the attribute/project name), and looking at it:

* Wut re:trailing slash for line continuation that doesn't continue
* Not wrapped properly: shebang isn't patched so fails since no /usr/bin/env in sandbox.

## Technical details

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the
results.

**Comment by dtzWill** [comment]
Okay that's what I thought/was hoping.  Looks like these (combined/squashed) do the trick:

https://github.com/dtzWill/nixpkgs/commit/60bf3e8d7b63632abb33d6cfcac02c191812699b
https://github.com/dtzWill/nixpkgs/commit/a5f2aafec316b7aa2a10c48439cf9e6c0fd712d7

**Comment by matthewbauer** [comment]
Yeah that looks right (patchShebangs shouldn't be needed, though, it's automatic).

It would be nice to be sure that perl isn't used as a native build input though. Could you try building with ```strictDeps = true```? That makes perl unavailable in the PATH of the builder.

**Comment by dtzWill** [comment]
Since the file is wrapped, patchShebangs misses the file `.parallel-wrapped' because it's hidden (I think)?

Not sure when will get back to this, sorry, but when I do I can try strictDeps as you suggest! :)

**Comment by matthewbauer** [comment]
That wasn’t an issue before though so i’d assume it can patch wrapped scripts.
